packages/runtime-core/__tests__/renderHelpers.spec.ts

```diff
@@ -0,0 +1,34 @@
+import Vue from '@vue/compat'
+import { PatchFlags, SlotFlags } from '@vue/shared'
+
+import { NULL_DYNAMIC_COMPONENT } from '../src/helpers/resolveAssets'
+import { toggleDeprecationWarning } from '../src/compat/compatConfig'
+import { createVNode } from '../src/vnode'
+
+import { legacyMarkOnce } from '../src/compat/renderHelpers'
+
+beforeEach(() => {
+  toggleDeprecationWarning(true)
+  Vue.configureCompat({
+    MODE: 2,
+    GLOBAL_MOUNT: 'suppress-warning',
+    GLOBAL_EXTEND: 'suppress-warning'
+  })
+})
+
+afterEach(() => {
+  toggleDeprecationWarning(false)
+  Vue.configureCompat({ MODE: 3 })
+})
+
+test('failling test', () => {
+  expect(1 + 2).toBe(3)
+})
```

Comment:
Is this bit supposed to be here? It looks like it might be left over from an earlier work-in-progress.